Exception in the fit() call of AutoSklearn

I am getting the following exception when running the very basic examples of AutoSklearn 1.0 and AutoSklearn 2.0. 
I have cloned the development branch and running the scripts on a Ubuntu machine with 128 GB RAM and 16 physical cores.
I am calling the classifier using the following lines; 

`model =  autosklearn.classification.AutoSklearnClassifier(time_left_for_this_task=5*60, per_run_time_limit=30, memory_limit=20240);
model.fit(X_train, y_train)
`

Traceback (most recent call last):
  File "autosklearnExp.py", line 42, in <module>
    model.fit(X_train, y_train)
  File "/home/user/auto-sklearn/autosklearn/estimators.py", line 1045, in fit
[ERROR] [2022-06-29 13:16:10,949:Client-AutoML(1):9fe4e2ba-f79c-11ec-b957-91a3ee291d16] Dummy prediction failed with run state StatusType.CRASHED and additional output: {'traceback': 'Traceback (most recent call last):\n  File "/home/user/auto-sklearn/autosklearn/evaluation/__init_                                                                       _.py", line 42, in fit_predict_try_except_decorator\n    return ta(queue=queue, **kwargs)\n  File "/home/user/auto-sklearn/autosklearn/eval                                                                       uation/train_evaluator.py", line 1195, in eval_holdout\n    evaluator.fit_predict_and_loss(iterative=iterative)\n  File "/home/user/auto-sk                                                                       learn/autosklearn/evaluation/train_evaluator.py", line 651, in fit_predict_and_loss\n    self.finish_up(\n  File "/home/user/auto-sklearn/a                                                                       utosklearn/evaluation/abstract_evaluator.py", line 365, in finish_up\n    file_out_loss, additional_run_info_ = self.file_output(\n  File "/hom                                                                       e/user/auto-sklearn/autosklearn/evaluation/abstract_evaluator.py", line 494, in file_output\n    self.backend.save_targets_ensemble(self.Y_                                                                       optimization)\nAttributeError: \'Backend\' object has no attribute \'save_targets_ensemble\'\n', 'error': 'AttributeError("\'Backend\' object h                                                                       as no attribute \'save_targets_ensemble\'")', 'configuration_origin': 'DUMMY'}.
    super().fit(
  File "/home/user/auto-sklearn/autosklearn/estimators.py", line 375, in fit
    self.automl_.fit(load_models=self.load_models, **kwargs)
  File "/home/user/auto-sklearn/autosklearn/automl.py", line 2056, in fit
    return super().fit(
  File "/home/user/auto-sklearn/autosklearn/automl.py", line 808, in fit
    self.num_run += self._do_dummy_prediction(datamanager, num_run=1)
  File "/home/user/auto-sklearn/autosklearn/automl.py", line 476, in _do_dummy_prediction
    raise ValueError(
ValueError: Dummy prediction failed with run state StatusType.CRASHED and additional output: {'traceback': 'Traceback (most recent call last):\                                                                       n  File "/home/user/auto-sklearn/autosklearn/evaluation/__init__.py", line 42, in fit_predict_try_except_decorator\n    return ta(queue=que                                                                       ue, **kwargs)\n  File "/home/user/auto-sklearn/autosklearn/evaluation/train_evaluator.py", line 1195, in eval_holdout\n    evaluator.fit_pr                                                                       edict_and_loss(iterative=iterative)\n  File "/home/user/auto-sklearn/autosklearn/evaluation/train_evaluator.py", line 651, in fit_predict_a                                                                       nd_loss\n    self.finish_up(\n  File "/home/user/auto-sklearn/autosklearn/evaluation/abstract_evaluator.py", line 365, in finish_up\n    fi                                                                       le_out_loss, additional_run_info_ = self.file_output(\n  File "/home/user/auto-sklearn/autosklearn/evaluation/abstract_evaluator.py", line                                                                        494, in file_output\n    self.backend.save_targets_ensemble(self.Y_optimization)\nAttributeError: \'Backend\' object has no attribute \'save_ta                                                                       rgets_ensemble\'\n', 'error': 'AttributeError("\'Backend\' object has no attribute \'save_targets_ensemble\'")', 'configuration_origin': 'DUMMY                                                                       '}.

--- a/test/fixtures/ensemble_building.py
+++ b/test/fixtures/ensemble_building.py
@@ -231,7 +231,10 @@ def make_ensemble_builder_manager(
             # Hence, we take the y_train of the datamanager and use that as the
             # the targets
             if "Y_train" in datamanager.data:
-                backend.save_targets_ensemble(datamanager.data["Y_train"])
+                backend.save_additional_data(
+                    datamanager.data["Y_train"],
+                    what="targets_ensemble",
+                )
 
         return EnsembleBuilderManager(
             backend=backend,

